apis/extension/descheduling.go

```diff
@@ -34,6 +37,37 @@ const (
 	AnnotationEvictionCost = SchedulingDomainPrefix + "/eviction-cost"
 )
 
+const (
+	// AnnotationSoftEviction indicates custom eviction. It can be used to set to an "true".
+	AnnotationSoftEviction = SchedulingDomainPrefix + "/soft-eviction"
+)
+
+type SoftEvictionSpec struct {
+	// Timestamp indicates time when custom eviction occurs . It can be used to set a second timestamp.
+	Timestamp int64 `json:"timestamp,omitempty"`
```

Comment:
It is recommended to change the type of the timestamp field to *metav1.Time so that it can be formatted as a time string. More readable than Unix Seconds.